gitpod-io/gitpod
# Non-project repository starting restrictions

## Description

Allow org owners to limit roles which can start workspaces from repositories that have not been imported.

## Related Issue(s)

Fixes ENT-816

## How to test


1. [Join my org](https://ft-org-resdabac04088.preview.gitpod-dev.com/orgs/join?inviteId=f287147a-2cb8-4f32-8193-413b76e7db8c)
2. Try to start any workspace
3. Import the same repo under Repository settings
4. Try it again (should work now)
